backend/gke/tfsetup.sh

```diff
@@ -17,31 +17,33 @@ git checkout "${CAP_TERRAFORM_BRANCH}"
 git pull
 
 # Clear out any existing variables file
-echo "" > terraform.tfvars.json
-yq merge --inplace --tojson --prettyPrint terraform.tfvars.json - <<HEREDOC
-project        : "$GKE_PROJECT"
-location       : "$GKE_LOCATION"
-node_pool_name : "$GKE_CLUSTER_NAME"
-instance_count : "$GKE_NODE_COUNT"
-preemptible    : "$GKE_PREEMPTIBLE"
-vm_type        : "UBUNTU"
-gke_sa_key     : "$GKE_CRED_JSON"
-gcp_dns_sa_key : "$GKE_DNSCRED_JSON"
-cluster_labels :
-    catapult-clustername: "$GKE_CLUSTER_NAME"
-    owner : "${OWNER}"
-cluster_name   : "$GKE_CLUSTER_NAME"
-k8s_version    : "latest"
+cat > terraform.tfvars.json <<HEREDOC
+{
+    "project"        : "$GKE_PROJECT",
+    "location"       : "$GKE_LOCATION",
+    "node_pool_name" : "$GKE_CLUSTER_NAME",
+    "instance_count" : "$GKE_NODE_COUNT",
+    "preemptible"    : "$GKE_PREEMPTIBLE",
+    "vm_type"        : "UBUNTU",
+    "gke_sa_key"     : "$GKE_CRED_JSON",
+    "gcp_dns_sa_key" : "$GKE_DNSCRED_JSON",
+    "cluster_labels" : {
+        "catapult-clustername": "$GKE_CLUSTER_NAME",
+        "owner" : "${OWNER}"
+    },
+    "cluster_name"   : "$GKE_CLUSTER_NAME",
+    "k8s_version"    : "latest"
+}
 HEREDOC
 
 if [ -n "${EXTRA_LABELS}" ] ; then
-    yq merge --inplace --tojson --prettyPrint --overwrite terraform.tfvars.json \
-        - <<< "cluster_labels: ${EXTRA_LABELS}"
+    jq --raw-output --argjson labels "${EXTRA_LABELS}" '.cluster_labels *= $labels' terraform.tfvars.json \
+        | sponge terraform.tfvars.json
 fi
 
 if [ -n "${GKE_INSTANCE_TYPE}" ] ; then
-    yq write --inplace --tojson --prettyPrint terraform.tfvars.json \
-        instance_type "${GKE_INSTANCE_TYPE}"
+    jq --argjson type "${GKE_INSTANCE_TYPE}" '.instance_type = $type' terraform.tfvars.json \
+        | sponge terraform.tfvars.json
```

Comment:
Perhaps we could we use a tmp file instead of adding a dependency on `moreutils`?

Reply:
Changed to a temporary file in 12c662930c67fd54d48a87b404f0b4f572d5ab99